utils/build_swift/driver_arguments.py

```diff
@@ -990,7 +989,8 @@ def create_argument_parser():
         action=arguments.action.enable)
     parser.add_argument(
         "--enable-tsan-runtime",
-        help="enable Thread Sanitizer on the swift runtime")
+        help="enable Thread Sanitizer on the swift runtime",
+        action=arguments.action.enable)
```

Comment:
please put help at the last argument here

Reply:
I'm not going to do this at the moment. I have that change in a follow-up PR to #12873